src/Sentry/SentryEvent.cs

```diff
@@ -164,6 +164,10 @@ public IReadOnlyList<string> Fingerprint
         /// <inheritdoc />
         public IReadOnlyDictionary<string, string> Tags => _tags ??= new Dictionary<string, string>();
 
+        internal bool HasException => Exception is not null || SentryExceptions?.Any() == true;
+
+        internal bool HasUnhandledException => SentryExceptions?.Any(x => !(x.Mechanism?.Handled ?? true)) ?? false;
```

Comment:
nit: At least use the first letter of what we're dealing with. `x` doesn't communicate much
```suggestion
        internal bool HasUnhandledException => SentryExceptions?.Any(e => !(e.Mechanism?.Handled ?? true)) ?? false;
```